utils/decodeMoves.js

```diff
@@ -0,0 +1,101 @@
+const { char2num } = require("./utils/char2num");
```

Comment:
You are already in the `./utils` folder. Correct import would be `./char2num`

Reply:
I really wonder if you ever try to run code before you open a pr meta, travis spotted this instantly.

Reply:
indeed, its been around 2 months (since time i moved to ubuntu 20.04, that didnt have npm support until a few days ago) that i stopped running gtp2ogs at all
but it will change soon now, sorry about that :+1:

Reply:
also i didnt know exports worked like that, now i see how it works :)

Reply:
Yes, imports work with relative path.

There are ways, at least in typescript, to also set some root folders so you can always refer them the same way. It's possible with some Js sompiles too i see, so you would get things like `~/utils/char2num` or, in TS when you set a path "utils" `@utils/char2num`.

Reply:
yes, i thought we were always referring to absolute path, but now i see :)